eth/tracers/api.go

```diff
@@ -487,7 +488,7 @@ func (api *API) traceBlock(ctx context.Context, block *types.Block, config *Trac
 	if config != nil && config.Reexec != nil {
 		reexec = *config.Reexec
 	}
-	statedb, err := api.backend.StateAtBlock(ctx, parent, reexec, nil)
+	statedb, err := api.backend.StateAtBlock(ctx, parent, reexec, nil, false)
```

Comment:
You want this and the others true :) Only traceChain should be false, shouldn't it?